lib/commands/module_create.js

```diff
@@ -271,6 +360,7 @@ CMD.prototype._createSkeleton = Promise.method(function() {
   actionTemplateJson.apiGateway.cloudFormation.Path = _this._module.name + '/' + _this._module.action;
   actionTemplateJson.apiGateway.cloudFormation.Method = 'GET';
   actionTemplateJson.apiGateway.cloudFormation.Type = 'AWS';
+  actionTemplateJson.lambda.cloudFormation.Runtime = _this._module.runtime;
```

Comment:
Probably belongs in the if-protected block just before the switch block below (line 366), as that detects if the module is a lambda method.